compiler/src/dmd/dmodule.d

```diff
@@ -442,6 +442,8 @@ extern (C++) final class Module : Package
     size_t nameoffset;          // offset of module name from start of ModuleInfo
     size_t namelen;             // length of module name in characters
 
+    bool isExplicitlyOutOfBinary;
```

Comment:
Need to update the corresponding header, also a short comment as above explaining what this is for. Also also, maybe put this below another bool like `bool insearch;1 if this is related to that, or `bool isPackageFile;`if it is not.

Reply:
When I went looking for a place to put it, nothing stood out as a good place. I.e. no bitfields.

Can move with extra space I suppose.

I did the automatic header update, but not the manual one, on amend will add.